### PR TITLE
add landing pages for each service

### PR DESCRIPTION
**Pull request by wei-lee**
@aidenkeating mind taking a look?

screen shot 2018-06-20 at 18 25 09
screen shot 2018-06-20 at 18 24 56
screen shot 2018-06-20 at 18 24 45
screen shot 2018-06-20 at 18 24 34

### CODE REVIEW

**Comment by aidenkeating** [comment]
👀

File: ios-showcase-template/root/RootInteractor.swift [hunk 1]
```diff
@@ -31,24 +31,81 @@ enum DocsURL: String {
...
     var router: RootRouter?
 
+    let idmServiceDescription = ServiceDesciption(header: "The identity service allows you to add authentication and authorization to your mobile app.", contentList: [
+        "Secure your mobile app using the industry standard OpenID Connect protocol",
+        "Add access control to your app based on user’s group membership",
+        "Easily implement SSO, multi factor authentication and Social Login support",
+        "Back end support for identity brokering and user federation"
+        ], footer: nil)
+    
+    let deviceSecDescription = ServiceDesciption(header: "The security service allows you to easily configure and manage device security, and trust checks for your mobile app", contentList: [
```
Comment (aidenkeating) [inline ios-showcase-template/root/RootInteractor.swift]: On Cordova this is

"The Device Security service..."

File: ios-showcase-template/root/RootInteractor.swift [hunk 2]
```diff
@@ -31,24 +31,81 @@ enum DocsURL: String {
...
+       "Perform a range of device trust checks on the mobile device, such as checking if the device is rooted, and allows you take proactive action based on the results",
```
Comment (aidenkeating) [inline ios-showcase-template/root/RootInteractor.swift]: "allows you to take"? It's the same on the Cordova app

File: ios-showcase-template/root/RootInteractor.swift [hunk 3]
```diff
@@ -31,24 +31,81 @@ enum DocsURL: String {
...
+        "Distribute SSL certificates with a mobile app to create a direct chain of trust (certificate pinning)"
+    ], footer: nil)
+    
+    let pushServiceDescription = ServiceDesciption(header: "The push notification service allows you send native push notifications to different mobile operating systems.", contentList: [
```
Comment (aidenkeating) [inline ios-showcase-template/root/RootInteractor.swift]: On Cordova this is

"The Push Notifications service allows..."

File: ios-showcase-template/root/RootInteractor.swift [hunk 4]
```diff
@@ -31,24 +31,81 @@ enum DocsURL: String {
...
+       "Code once and send push notifications to iOS and Android",
+       "Push notifications to defined groups",
+       "Push notifications to either iOS only or Android only",
+       "Push notifications to different variants of a mobile app",
+       "Push notifications from different back-end apps to the same mobile app"
+    ], footer: nil)
+    
+    let metricsServiceDescription = ServiceDesciption(header: "The metrics service allows you gather metrics on mobile apps, device versions, device security checks and back-end mobile service usage.", contentList: [
```
Comment (aidenkeating) [inline ios-showcase-template/root/RootInteractor.swift]: On Cordova this is

"The Mobile Metrics service allows..."

**Review by aidenkeating — COMMENTED** [review]
@wei-lee Few small comments. I was comparing to the Cordova screenshots.

Comparing it with the Cordova app, I think we can make the font sizes a bit larger, especially on the bullet pointed lists.

File: ios-showcase-template/root/RootInteractor.swift (hunk 2 above)
Comment (wei-lee) [reply ios-showcase-template/root/RootInteractor.swift]: It's from the doc: https://docs.google.com/document/d/18aN0VqLy9BaFGqzjRSzPBAJC8Q-YkuiMp_dRMY_b5cw/edit.

cc @StephenCoady

**Comment by wei-lee** [comment]
screen shot 2018-06-21 at 10 07 54

**Comment by wei-lee** [comment]
@aidenkeating I have addressed your comments now. Mind taking another look?

**Review by aidenkeating — APPROVED** [review]
Looks good 👍

**Pull request merged**